Presentation.Web/Controllers/API/EconomyStreamController.cs

```diff
@@ -30,7 +30,7 @@ public HttpResponseMessage GetExternEconomyStreamForContract(int externPaymentFo
                     result = result.Where(x => x.AccessModifier == AccessModifier.Public || x.ExternPaymentFor.OrganizationId == currentOrgId);
                     if (!result.Any())
                         //at this point the economy streams are marked Local but the user is not part of the organization which means they are not authorized to view the data
```

Comment:
Ændrer kommentaren til at sige: ... not allowed to view the data

Reply:
Eller slet den helt

Reply:
Rettet til not allowed